src/track/bpm.h

```diff
@@ -25,6 +25,12 @@ class Bpm final {
 
     // Adjusts floating-point values to match their string representation
     // in file tags to account for rounding errors.
+    // NOTE(2019-02-19, uklotzde): The pre-normalization cannot prevent
```

Comment:
Why can't we handle this case?
Do we actually repetitive write metadata if we detect 128.1 but 128 is stored?

Reply:
ID3v2 is restricted to integer precision.

Reply:
For other tag formats the internal 64-bit (binary) floating point value is stored as a (decimal) string. This will cause rounding errors that we need to compensate for. Otherwise the metadata is always detected as modified even if the file tags are effectively unchanged.

Reply:
The critical point is: Is a 128.1 BMP track repeatedly written, because the decimal place is lost. 
I don't think this is the case, but the comment implies that.

Reply:
That is in fact true. I added a TL;DR to emphasize this point!

We can't do anything about it if we don't want to reduce the precision for the bpm comparison in general. This would be the only option.

Reply:
I have switched the corresponding *modified* check to integer precision comparison. As a minor drawback fractional bpm changes are not detected as modified. But we avoid modifying many files that actually don't need to be modified.

Just recently I saw complaints by Serato users, because Serato is writing library details like the *play count* into their marker tags! Every unmodified file you play needs to be copied during the next backup and this strategy also reduces the lifetime of your SSD.

Reply:
Sounds reasonable. Thank  you.